src/Components/Blueprints/ImportBlueprintModal.tsx

```diff
@@ -33,69 +36,83 @@ export const ImportBlueprintModal: React.FunctionComponent<
   const onImportClose = () => {
     setShowImportModal(false);
     setFilename('');
-    setJsonContent('');
+    setFileContent('');
     setIsRejected(false);
     setIsInvalidFormat(false);
   };
-  const [jsonContent, setJsonContent] = React.useState('');
+  const [fileContent, setFileContent] = React.useState('');
   const [importedBlueprint, setImportedBlueprint] =
     React.useState<wizardState>();
   const [isInvalidFormat, setIsInvalidFormat] = React.useState(false);
   const [filename, setFilename] = React.useState('');
   const [isLoading, setIsLoading] = React.useState(false);
   const [isRejected, setIsRejected] = React.useState(false);
+  const [isOnPrem, setIsOnPrem] = React.useState(false);
```

Comment:
You might want to reset this flag in `onClear` handler.

Reply:
Good catch, thank you! I added more resets to the default value, you are right.